ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController_SwiftUI.swift

```diff
@@ -50,6 +52,7 @@ struct AvatarDemoView: View {
                 .imageBasedRingColor(showImageBasedRingColor ? AvatarDemoController.colorfulCustomImage : nil)
                 .isTransparent(isTransparent)
                 .presence(presence)
+                .activity(activityStyle, showActivityImage ? UIImage(named: "excelIcon") : nil)
                 .isOutOfOffice(isOutOfOffice)
                 .hasPointerInteraction(hasPointerInteraction)
                 .isAnimated(isAnimated)
```

Comment:
I can't quite get to it here, but can we change the `Color.gray` to like `Colors.surfacePrimary`, or something? Or an alias token.